templ_enum.cif

```diff
@@ -870,49 +870,32 @@ save_atomic_number
      loop_
```

Comment:
Shouldn't the atomic mass also depend on the mass number? Maybe the changes to this frame/item should be delayed until we can actually specify the multivalue key?

Reply:
Yes, it should.